<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Sarfaraj_Doni" name="courses.BoatEd.MinnesotaTest" tests="8" failures="0" timestamp="2 Jan 2017 16:27:04 GMT" time="2888.153" errors="1">
  <testcase name="MN_Navigate_To_StudyGuide_Page" time="117.496" classname="courses.BoatEd.MinnesotaTest"/>
  <testcase name="MN_TimedCourseVerification" time="257.766" classname="courses.BoatEd.MinnesotaTest"/>
  <testcase name="MN_VerifyFeeAndTaxes" time="356.360" classname="courses.BoatEd.MinnesotaTest"/>
  <testcase name="MN_Stds_Req_International_Address" time="349.249" classname="courses.BoatEd.MinnesotaTest"/>
  <testcase name="MN_VerifyAgencyName" time="109.709" classname="courses.BoatEd.MinnesotaTest"/>
  <testcase name="MN_Stds_Req_AgeLimit" time="227.515" classname="courses.BoatEd.MinnesotaTest"/>
  <testcase name="MN_VerifyPraticeExam" time="1140.367" classname="courses.BoatEd.MinnesotaTest">
    <error type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.fail(Assert.java:101)
	at courses.BoatEd.MinnesotaTest.MN_VerifyPraticeExam(MinnesotaTest.java:1225)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:57)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:606)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:84)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
]]>
    </error>
  </testcase> <!-- MN_VerifyPraticeExam -->
  <testcase name="MN_AskedForPaymentAfterPassingExam" time="329.691" classname="courses.BoatEd.MinnesotaTest"/>
</testsuite> <!-- courses.BoatEd.MinnesotaTest -->
